ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -15,6 +15,9 @@ images:
       as:
       - node:15
   to: odf-console
+promotion:
+  name: "4.12"
+  namespace: ocp
```

Comment:
Do we need this? Shouldn't this change be in [`releases.integration`](https://docs.ci.openshift.org/docs/getting-started/examples/#how-do-i-use-an-image-from-another-repo-in-my-repos-tests) instead?

Reply:
Like so: https://github.com/openshift/release/pull/32612/files.

cc @SanjalKatiyar (your PR can be closed if @vbnrh makes this change in this PR itself).

Reply:
This is not for changing ocp version to 4.12.
I had added them for automated branching but can be removed for now.
https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching